spec/unit/bricks/middleware/logger_middleware_spec.rb

```diff
@@ -52,4 +52,36 @@
       end
     end
   end
+
+  context 'when COLLECT_STATS parameter set to true' do
```

Comment:
Similar code found in logger_middleware_spec.rb:56, logger_middleware_spec.rb:72 (mass = 90)